explorer/src/components/Storage/APIs/swagger.json

```diff
@@ -0,0 +1 @@
earchResult":{"type":"object","properties":{"cid":{"type":"string"},"name":{"type":"string"}},"required":["cid","name"]},"OffchainMetadata":{"type":"object","oneOf":[{"$ref":"#/components/schemas/OffchainFileMetadata"},{"$ref":"#/components/schemas/OffchainFolderMetadata"}]},"OffchainFileMetadata":{"type":"object","properties":{"type":{"type":"string","enum":["file"]},"dataCid":{"type":"string"},"name":{"type":"string","nullable":true},"mimeType":{"type":"string","nullable":true},"totalSize":{"type":"string"},"totalChunks":{"type":"integer"},"chunks":{"type":"array","items":{"$ref":"#/components/schemas/ChunkInfo"}},"uploadOptions":{"$ref":"#/components/schemas/UploadOptions"}}},"ChunkInfo":{"type":"object","properties":{"size":{"type":"string"},"cid":{"type":"string"}}},"OffchainFolderMetadata":{"type":"object","properties":{"type":{"type":"string","enum":["folder"]},"dataCid":{"type":"string"},"name":{"type":"string","nullable":true},"totalSize":{"type":"string"},"totalFiles":{"type":"integer"},"children":{"type":"array","items":{"$ref":"#/components/schemas/ChildrenMetadata"}},"uploadOptions":{"$ref":"#/components/schemas/UploadOptions"}}},"ChildrenMetadata":{"type":"object","properties":{"type":{"type":"string","enum":["file","folder"]},"cid":{"type":"string"},"name":{"type":"string","nullable":true},"totalSize":{"type":"string"}},"required":["type","cid","totalSize"]},"UploadStatus":{"type":"object","properties":{"uploadedNodes":{"type":"integer","nullable":true},"totalNodes":{"type":"integer","nullable":true},"archivedNodes":{"type":"integer","nullable":true},"minimumBlockDepth":{"type":"integer","nullable":true},"maximumBlockDepth":{"type":"integer","nullable":true}}},"UploadOptions":{"type":"object","properties":{"compression":{"$ref":"#/components/schemas/CompressionOptions"},"encryption":{"$ref":"#/components/schemas/EncryptionOptions"}}},"CompressionOptions":{"type":"object","properties":{"algorithm":{"type":"string","enum":["ZLIB"]},"level":{"type":"integer","default":8}}},"EncryptionOptions":{"type":"object","properties":{"algorithm":{"type":"string","enum":["AES_256_GCM"]},"chunkSize":{"type":"integer","default":1024}}},"SubscriptionInfo":{"type":"object","properties":{"id":{"type":"string"},"organizationId":{"type":"string"},"uploadLimit":{"type":"number"},"downloadLimit":{"type":"number"},"granularity":{"type":"string","enum":["monthly"]},"pendingUploadCredits":{"type":"number"},"pendingDownloadCredits":{"type":"number"}},"required":["id","organizationId","uploadLimit","downloadLimit","granularity","pendingUploadCredits","pendingDownloadCredits"]}},"securitySchemes":{"apiKey":{"type":"apiKey","in":"header","name":"Authorization","description":"Bearer token for authentication. Example: Bearer <token>. Could be either an API key or a JWT token."},"provider":{"type":"apiKey","in":"header","name":"X-Auth-Provider","description":"Used for differentiating between different auth providers. For most use cases: X-Auth-Provider: apikey"}}}}
```

Comment:
Is there a reason for hardcoding the Swagger API into astral instead of retrieving the spec from:

https://demo.auto-drive.autonomys.xyz/api/docs/raw
https://mainnet.auto-drive.autonomys.xyz/api/docs/raw